### PR TITLE
chore: typography cleanup

### PR DESCRIPTION
**Pull request by hofmann-felix**
### What's included?
* The font sizes of the content are now consistent:
  * Mobile: `16px`
  * Desktop: `18px`
 
Preview URL: https://satellytescomfeatmaindesignupd-choretypographyupdate.gatsbyjs.io/de/

Preview of the old (broken) version: https://satellytescomfeatmaindesignupd.gatsbyjs.io/de/

### CODE REVIEW

**Review by georgiee — COMMENTED** [review]
I've issues following that change and where to look at. Can you help?

Also  while being here I checked the headlines (because I though it's about them from the title).

I use an extension to quickly check the outline of headlines. Looks great on most pages but on the career page we have two times `h1`. Plus the sizes of the two `h1` are different (which makes it a topic relevant to post here). Maybe on thing missing in the cleanup?

Screenshot 2023-03-21 at 11 48 54

Screenshot 2023-03-21 at 11 48 05


Neither approval or change requested so far. I hope you can clarify the current changes and give feedback to my finding. Thanks!

**Comment by hofmann-felix** [comment]
@georgiee Currently there are still some places on the page where the font size is not consistent. I took a few screenshots  of the inconsistent texts to make the changes better understandable: 

**Desktop Service Page**
Bildschirm­foto 2023-03-21 um 14 28 47

**Mobile Career Form**
Bildschirm­foto 2023-03-21 um 14 44 57

**Mobile Landing Page**
Bildschirm­foto 2023-03-21 um 14 42 09

**Mobile Career Page**
Bildschirm­foto 2023-03-21 um 14 41 10

**Desktop Career Page**
Bildschirm­foto 2023-03-21 um 14 38 07

File: src/components/forms/file-dropper/file-dropper.tsx [hunk 1]
```diff
@@ -126,6 +126,8 @@ const Description = styled.p`
   ${TextStyles.timestamp};
   letter-spacing: 0;
   margin: 0;
+  padding: 0 24px;
+  text-align: center;
```
Comment (hofmann-felix) [inline src/components/forms/file-dropper/file-dropper.tsx:130]: While making the screenshots, I noticed that padding and text align center are missing here on mobile

Bildschirm­foto 2023-03-21 um 15 06 20

**Review by georgiee — APPROVED** [review]
Thanks for the clarification. Let's merge it 🙌

**Pull request merged**